File: rentabikewtr_mobile/lib/model/poziviDezurnomVozilu.dart
//import 'dart:ffi';

import 'package:json_annotation/json_annotation.dart';

part 'poziviDezurnomVozilu.g.dart';

@JsonSerializable()
class PoziviDezurnomVozilu {
  String? viseDetalja = "";
  bool? nesreca;
  bool? zahtjevKlijenta;
  bool? kvar;
  bool? losiVremenskiUslovi;
  DateTime? datumPoziva = DateTime.now();
  DateTime? vrijemePoziva;
  int? dezurnoVoziloID;
  int? turistickiVodicID;
  int? poslovnicaID;

  PoziviDezurnomVozilu(
      {this.viseDetalja,
      this.nesreca,
      this.zahtjevKlijenta,
      this.kvar,
      this.losiVremenskiUslovi,
      this.datumPoziva,
      this.vrijemePoziva,
      this.dezurnoVoziloID,
      this.turistickiVodicID,
      this.poslovnicaID}) {}

  factory PoziviDezurnomVozilu.fromJson(Map<String, dynamic> json) =>
      _$PoziviDezurnomVoziluFromJson(json);

  /// Connect the generated [_$PoziviDezurnomVoziluToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$PoziviDezurnomVoziluToJson(this);
}

File: RentABikeWTR_v1_7.Model/PoziviDezurnomVozilu.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace RentABikeWTR_v1_7.Model
{
    public class PoziviDezurnomVozilu
    {
        public int? PozivDezurnomVoziluId { get; set; }
        public string? ViseDetalja { get; set; }
        public bool? Nesreca { get; set; }
        public bool? ZahtjevKlijenta { get; set; }
        public bool? Kvar { get; set; }
        public bool? LosiVremenskiUslovi { get; set; }

        public DateTime? DatumPoziva { get; set; }
        public DateTime? VrijemePoziva { get; set; }

        public int? DezurnoVoziloID { get; set; }
        public DezurnaVozila? DezurnoVozilo { get; set; }

        public int? TuristickiVodicID { get; set; }
        public virtual TuristickiVodici TuristickiVodic { get; set; }

        public int? PoslovnicaID { get; set; }

        public virtual Poslovnice Poslovnica { get; set; } = new Poslovnice();
    }
}
